### DISCUSSION TITLE
Obter o lucro das operações

### DISCUSSION THREAD

**Discussion by mvstec** [RamonSilva20/mapos#2368] [Ideas]
Atualmente as operações de vendas não refletem a MARGEM DE LUCRO no relatorio de vendas, pois é de muito importância essa métrica para o empresário poder ter noções de faturamento líquido, sem a necessidade de outros sistemas ou planilhas externas.

**Comment by Marc3lo-sa** [comment]
O projeto é bem estruturado, mas eu quase não acreditei que está faltando  essa parte do lucro das vendas de produtos e serviços .
